profile/README.md

```diff
@@ -1,56 +1,34 @@
-  
-</details>
-
-## Jahia Academy: our documentation site
-
-Most of our documentation, including developer documentation, may be found on the [Jahia Academy](https://academy.jahia.com).
-
-<details>
-
-- [Get Started with Jahia](https://academy.jahia.com/get-started) : All you need to get up and running with your first Jahia install and initial development
-- [Downloads](https://academy.jahia.com/home/downloads.html) : All the downloads of the latest versions of Jahia, as well as a link to our Customer Center for additional downloads for existing customers
-- [Developer documentation](https://academy.jahia.com/documentation/developer) : All the developer documentation
-- [Developer Webinars](https://academy.jahia.com/customer-center/webinars-for-developers) : A collection of past webinars to get the latest and greatest about developing with Jahia
-- [Online developer training](https://academy.jahia.com/training-kb/online-developer-training#tab-66634213-8154-4a3d-8d43-d728f57ac094) : This free training introduces developer basics and shows how to develop templates. The training usually takes 2 days to complete.
-
-## Connect with us
-
-Don't hesitate to reach us by using our [Contact Page](https://www.jahia.com/contact) or if you are an existing customer, you can use our [Support Issue tracker](https://support.jahia.com)
-
-Also, you can find us on  
-[LinkedIn](https://www.linkedin.com/company/jahia-solutions/) <a href="https://www.linkedin.com/company/jahia-solutions/"><img align="left" src="https://raw.githubusercontent.com/yushi1007/yushi1007/main/images/linkedin.svg" alt="Jahia Solutions | LinkedIn" width="21px"/></a>
-</details>
+<a href="https://www.jahia.com/en">
+  <img src="./banner.png" alt="Jahia: Open Source Technologies for Enterprise Websites"/>
+</a>
+
+### Learn Jahia
+
+<table>
+  <thead>
+    <tr>
+      <th>For JS Developers</th>
+      <th>For Java Developers</th>
+      <th>For Content Editors</th>
+      <th>Knowledge Base</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/Jahia/javascript-modules/blob/main/docs/1-getting-started/README.md">Build a  React Website with Jahia</a></td>
+      <td><a href="https://academy.jahia.com/training-kb/online-developer-training#developer-training">Build a JSP Website with Jahia</a></td>
+      <td><a href="https://academy.jahia.com/training-kb/online-developer-training#editor-training">Learn Jahia for Content Editors</a></td>
+      <td><a href="https://academy.jahia.com/">Jahia Academy</a></td>
+    </tr>
+  </tbody>
+</table>
+
+<p align="center">
+  <small>
+    <a href="https://support.jahia.com/secure/Dashboard.jspa">Customer Support</a>
+    • <a href="https://github.com/Jahia/jahia/issues">Report a Bug</a>
+    • <a href="https://www.jahia.com/contact">Contact Us</a>
+    • <a href="https://hub.docker.com/u/jahia">Docker Hub</a>
```

Comment:
I'd put "Docker Images" instead of "Docker Hub" here.

Reply:
```suggestion
    • <a href="https://hub.docker.com/u/jahia">Docker Images</a>
```